centaur/src/main/resources/standardTestCases/drs_usa_hca/drs_usa_hca.wdl

```diff
@@ -0,0 +1,53 @@
+version 1.0
+
+workflow drs_usa_hca {
+    call localize_drs_with_usa
+
+    output {
+        String hash1 = localize_drs_with_usa.hash1
+        String hash2 = localize_drs_with_usa.hash2
+        String hash3 = localize_drs_with_usa.hash3
+        String hash4 = localize_drs_with_usa.hash4
+        String hash5 = localize_drs_with_usa.hash5
+        Float size1 = localize_drs_with_usa.size1
+        Float size2 = localize_drs_with_usa.size2
+        Float size3 = localize_drs_with_usa.size3
+        Float size4 = localize_drs_with_usa.size4
+        Float size5 = localize_drs_with_usa.size5
+    }
+}
+
+task localize_drs_with_usa {
+    # For all below 5 HCA uuids, Martha does not return a service account
+    File file1 = "dos://service.staging.explore.data.humancellatlas.org/033c9840-c5cd-438b-b0e4-8e4cd8fc8dc6?version=2019-07-04T104122.106166Z"
```

Comment:
TOL: Might be nice to pass these files in as inputs rather than hard-code them in WDL